minions.md

```diff
@@ -144,6 +146,22 @@ Agility rewards you with marks of grace which you can use to buy Graceful equipm
 
 Click [here](https://i.imgur.com/LZGlSgj.png) for **Agility** XP rates and **Marks of Grace** per hour.
 
+#### Monkey Backpacks
+
+When completing laps of the Ape Atoll agility course you can earn a [monkey backpack](https://oldschool.runescape.wiki/w/Monkey_(Monkey_Madness_II))! These are obtained automatically whenever you reach the lap count required for each backpack.
+
+| **Monkey** | **Required laps** |
+| - | :-: |
+| Karamjan | 100 |
+| Zombie | 250 |
+| Maniacal | 500 |
+| Skeleton | 1,000 |
+| Kruk jr. | 1,500 |
+| Princley | 2,000 |
+
+To view your total lap count on each specific agility course you can use the `+lap` command (note: only counts laps performed since August 3rd, 2020). 
```

Comment:
```suggestion
To view your total lap count on each specific agility course you can use the `+laps` command (note: only counts laps performed since August 3rd, 2020). 
```